bin/chroma_server.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+docker run --rm -it -p 8000:8000 -e ALLOW_RESET=TRUE chromadb/chroma:latest
```

Comment:
maybe let's move this under `scripts/chroma_server.sh` 

usually we place some binaries built in go under `bin/xxx`

Reply:
good point